deploy/sre-prometheus/100-ocm-agent-operator.PrometheusRule.yaml

```diff
@@ -38,3 +38,15 @@ spec:
         link: "https://github.com/openshift/ops-sop/blob/master/v4/alerts/OCMAgentResponseFailureServiceLogsSRE.md#verify-cluster-pull-secrets"
       annotations:
         message: OCM Agent pull secret auth token is not valid.
+
+
+    # Alert SRE if the OCM agent service log metric has been sent more than 10 within a 3-day period.
+    - alert: OCMAgentServiceLogsSentExceedingLimit
+      expr: sum(increase(ocm_agent_service_log_sent_total[3d])) > 10
```

Comment:
One thing that might cause unwanted pages: we have ocm-agent-operator running on the centralized observability cluster in `fleet-mode` and sending the service logs for all hypershift clusters. 
While 10 service logs is a lot to send to a single cluster (when running in standard mode), it's a pretty low number when looking at the whole fleet.

**Suggestion:** we could scope the alert expression per `_id` which represents the cluster identifier, so that it works the same way for fleet mode and standard mode.

Reply:
We don't seem to be adding `_id` in the `ocm_agent_service_log_sent_total` metric.

@ravitri / @bmeng - probably something we want to figure out with this alert. We need to make sure it works for fleet-mode as well as non-fleet mode. Do you think we should be adding `_id` to the `ocm_agent_service_log_sent_total` metric, then we could do something like:
```
sum(increase(ocm_agent_service_log_sent_total[3d])) by (_id) > 10
```

Reply:
The function `SetTotalServiceLogCount` which is setting the metrics is not being called by the fleetmode webhook for now.
I feel it would be easier if we just define different metrics if we would like to cover fleet mode

Reply:
That makes sense @bmeng, thanks.